### DISCUSSION TITLE
求助 挂载115失败

### DISCUSSION THREAD

**Discussion by Lenzzx** [AlistGo/alist#3994] [Q&A]
用的是斐讯N1 openwrt alist插件挂载115网盘
cookie和根文件id输入了以后提示
Failed init storage: Get "https://passportapi.115.com/app/1.0/web/1.0/check/sso?_=1676354142978": dial tcp: lookup passportapi.115.com on [**1]:53: server misbehaving
用windows版挂载115没有提示这个问题

**Comment by Rektyfikowany** [comment]
检查openwrt上的DNS设置，从盒子ping passportapi.115.com能否ping通
